sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/NamedVIPSpec.java

```diff
@@ -84,7 +87,8 @@ public ResourceRequirement getResourceRequirement(Protos.Resource resource) {
                 getProtocol(),
                 getVisibility(),
                 getVipName(),
-                getVipPort());
+                getVipPort(),
+                super.getNetworkNames());
```

Comment:
Should either omit the `super.` here, or add `super.` to other cases where it's coming from the parent class (eg `getPortName()`, `getValue()`, ...). I'd actually lean towards the latter tbh